var/spack/repos/builtin/packages/py-pacifica-namespace/package.py

```diff
@@ -0,0 +1,17 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyPacificaNamespace(PythonPackage):
+    """Python Pacifica Namespace Library"""
+
+    homepage = "https://github.com/pacifica/pacifica-namespace/"
+    pypi     = "pacifica-namespace/pacifica-namespace-0.0.2.tar.gz"
+
+    version('0.0.2', sha256='a8f59aea1203a1557f7f57887b61e92f8450c74a8522798c5ddecf8fffb0b224')
+
+    depends_on('python@3:', type=('build', 'run'))
```

Comment:
Where do you see this? Also, you're missing a dep on setuptools and possible setuptools_scm

Reply:
i added setuptools

Reply:
I would suggest dropping this because the supported Python versions aren't explicitly enforced in the setup.py.

Reply:
Also need build dep on `py-setuptools-scm`

Reply:
changed